src/components/Table/TableHead/state/sorting.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { sortRows } from '../tools/sorting';
+
+/**
+ * We currently support the following sorting states for DataTable headers,
+ * namely: `NONE` for no sorting being applied, and then `DESC` and `ASC` for
+ * the corresponding direction of the sorting order.
+ */
+export const sortStates = {
```

Comment:
I don't think we need to bring this whole file in. Let's copy `sortStates` over in the `TableHeader.js` file that uses it.

Reply:
Done